bundles/com.espressif.idf.core/src/com/espressif/idf/core/tools/EimConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.espressif.idf.core.tools;
+
+public interface EimConstants
+{
+	String EIM_WIN_PATH = "C:\\Espressif\\tools\\esp_ide.json"; //$NON-NLS-1$
```

Comment:
For some reason, if user is not installed in C:\Espressif\ folder, do we have other mechanism to identify this file?

Reply:
Not sure how to handle this since I believe this should be constant. I will try to discuss this with @[Hahihula](https://github.com/Hahihula)

Reply:
Right now we do not enable to user to configure this location. In the future we can store the value in registry on windows sopmewhere under 'HKEY_CURRENT_USER' but on other system ( like linux) we would probably still rely on constant file location.
The only fallback I can think of is to (if the files does not exist) offer user to manually locate it.